code/__defines/_unit_tests.dm

```diff
@@ -1,7 +1,7 @@
 //include unit test files in this module in this ifdef
 //Keep this sorted alphabetically
 
-// #if defined(UNIT_TESTS) || defined(SPACEMAN_DMM) //tgstation style, not relevant for us, for now
+// #if defined(UNIT_TEST) || defined(SPACEMAN_DMM) //tgstation style, not relevant for us, for now
```

Comment:
if this is commented out code and not a comment, it should be removed.

Reply:
I think I have left it to remind me that everything down there is supposed to not be included if it's not a unit test or a linter, the problem is that we never separated things before so it was not possible to just not include it until everything is brought to not rely on those defines, either way it's currently a spell fix and that acts as a comment